org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/ConsistencyCheckErrors.java

```diff
@@ -0,0 +1,63 @@
+package org.eclipse.winery.repository;
+
+import org.eclipse.winery.common.ids.definitions.DefinitionsChildId;
+
+public class ConsistencyCheckErrors {
+
+	private DefinitionsChildId checkedId;
+	private DefinitionsChildId inconsistentId;
+
+	private boolean biggerError;
+	
+	private String biggerErrorMessage = "";
+	
+	public ConsistencyCheckErrors() {
+		biggerError = false;
+	}
+
+	@Override
+	public String toString() {
+		if (this.isBiggerError()) {
+			return this.getBiggerErrorMessage();
+		} 
+		return getErrorMessageForIds();
+	}
+
+	private String getErrorMessageForIds() {
+		return "Found error by " + this.getCheckedId().getClass().getCanonicalName() 
+			+ "checking id" + this.getCheckedId().getQName() + " with inconsistend Id " 
+			+ this.getInconsistentId().getQName();
+	}
+
+	public DefinitionsChildId getCheckedId() {
+		return this.checkedId;
+	}
+
+	public void setCheckedId(DefinitionsChildId checkedId) {
+		this.checkedId = checkedId;
+	}
+
+	public DefinitionsChildId getInconsistentId() {
+		return this.inconsistentId;
+	}
+
+	public void setInconsistentId(DefinitionsChildId inconsistentId) {
+		this.inconsistentId = inconsistentId;
+	}
+
+	public boolean isBiggerError() {
```

Comment:
Please comment on the difference between "bigError" and "smallError"

Please rename "bigger" to "big".